### DISCUSSION TITLE
query

### DISCUSSION THREAD

**Discussion by saakshii12** [arjuncvinod/Hello-World-in-Different-Languages#130] [General]
my PR were not accepted on hactoberfest main website as your repository is excluded from hacktoberfest , so what can i do now?

**Comment by Ankitjarwall** [comment]
To get your pull requests (PRs) accepted for Hacktoberfest, you need to ensure that the repositories you contribute to meet the Hacktoberfest requirements. Here are some steps you can take:

Check Repository Status:

Make sure the repository you are contributing to has the hacktoberfest topic added to it. This is a requirement for it to be considered for Hacktoberfest.
The repository should not be excluded from Hacktoberfest. Check the [list of excluded repositories](https://github.com/digitalocean/hacktoberfest-repository-eligibility) to ensure the repository you are contributing to is not on it.
Choose Eligible Repositories:

Look for repositories that are actively participating in Hacktoberfest. You can find these by searching for repositories with the hacktoberfest topic on GitHub.
You can also check the Hacktoberfest website for a list of participating repositories.
Follow Contribution Guidelines:

Make sure to follow the contribution guidelines of the repository you are contributing to. This includes making meaningful contributions and not just spammy or low-quality PRs.
Get Maintainer Approval:

Ensure that your PRs are reviewed and accepted by the repository maintainers. Only merged PRs or those with the hacktoberfest-accepted label count towards Hacktoberfest.
Participate in Hacktoberfest Events:

Join Hacktoberfest events and meetups to find repositories and network with other contributors and maintainers.


hope this could help you, Thank you
